dialects/clos/pass/rw/clos2sjlj.cpp

```diff
@@ -87,9 +87,9 @@ Lam* Clos2SJLJ::get_throw(const Def* dom) {
         auto [m0, env, var]    = split(tlam->var());
         auto [jbuf, rbuf, tag] = env->projs<3>();
         auto [m1, r]           = mem::op_alloc(var->type(), m0)->projs<2>();
-        auto m2                = mem::op_store(m1, r, var);
+        auto m2                = w.call<mem::store>(Defs{m1, r, var});
```

Comment:
Same logic as with `World::ax`. If you have an axiom with subtags, you invoke like this:
```c++
w.call<core::wrap>(core::wrap::add, 0, Defs{a, b});
```
or - since `core::wrap` will be automatically inferred by C++ - you can elide the template arg:
```c++
w.call(core::wrap::add, 0, Defs{a, b});
```
If you don't have a subtag, you *must* provide the template parameter:
```c++
w.call<mem::store>(Defs{m, ptr, val})
```